README.md

```diff
@@ -17,6 +17,10 @@ If you do not, or have already run the above, run:
 
     GOEXPERIMENT=arenas go run ./...
 
+## Beginners' Guide to Setup
+
+Here is a detailed guide for non-technical folks who want to setup a node: [link](https://demipoet.github.io/)
```

Comment:
I understand completely. I will retract this pull request, and potentially only embed the detailed guide itself in the main repo, rather than link it externally.